src/ASM/LR/ASMu2Dmxrecovery.C

```diff
@@ -21,6 +21,7 @@
 #include "SparseMatrix.h"
 #include "SplineUtils.h"
 #include "Profiler.h"
+#include "matrix.h"
```

Comment:
Can not see why you need to add this include here (it is included via Sparsematirx.h anyway)

Reply:
removed.